UIKitHw/Controllers/TimerViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  TimerViewController.swift
+//  UIKitHw
+//
+//  Created by Aleksandr Dorofeev on 05.10.2022.
+//
+
+import UIKit
+
+/// Timer screen.
+class TimerViewController: UIViewController {
+  
+  // MARK: - Private enum.
+  private enum Constants {
+    static let timerCounterStartPosition = 0.00
+    static let interval = 0.01
+    static let customRedColor = UIColor(red: 0.666, green: 0.335, blue: 0.338, alpha: 1)
+    static let customGreenColor = UIColor(red: 0.486, green: 0.666, blue: 0.459, alpha: 1)
+    static let stopTitle = "Стоп"
+    static let startTitle = "Старт"
+    static let startPositionLabel = "0:00:00"
```

Comment:
```suggestion
    static let startPosition = "0:00:00"
```
Label пишем если тип Label